src/components/common/QuestionWrapper.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <section class="mt-5" :id="id">
+    <header>
+      <h2>{{ title }}</h2>
+      <p v-if="subtitle" class="pt-2">{{ subtitle }}</p>
+      <p v-if="comment" class="black--text mb-0">{{ comment }}</p>
+    </header>
+    <slot></slot>
+  </section>
+</template>
+
+<script>
+export default {
```

Comment:
What do you think of making this component to be shared sometime in the future? Since the same thing is being used also in Create UI.